### ISSUE TITLE
Plugin breaks "Edit Configurations" (Run Configuration)

### ISSUE THREAD

**Issue by craftey** [datawire/telepresence-plugin-jetbrains-issues#2]
If telepresence binary is not installed, existing Run Configurations can be edited fine.
Also we get a notification that informs us to install telepresence.
```
# On macOS
brew unlink telepresence-arm64
```

------------

If the wrong v1 binary is installed, existing Run Configurations cannot be edited fine.
No informing notification. This is hard to track down.
```
brew unlink telepresence-arm64 ; brew link telepresence-legacy # tele1
```
![image](https://github.com/datawire/telepresence-plugin-jetbrains-issues/assets/17045396/0c6a04d3-d03d-40b2-95ed-2c06d0c05d72)
We just see a blank page for any existing run config. New run configs cannot be created.

-----------

If we install the correct version v2 than Run Configs work fine again:
```
brew unlink telepresence-legacy ; brew link telepresence-arm64 # tele2
```

-----------

TLDR:
Please guard against wrong installed telepresence-legacy.
